File: comms/push-notifications/credentials/update-credential/update-credential.go
package example

import (
    context "context"

    twiliocomms "github.com/twilio/twilio-comms-go/twilio"
    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "TWILIO_ACCOUNT_SID",
            "TWILIO_AUTH_TOKEN",
        ),
    )
    request := &twiliocomms.CredentialsUpdateRequest{
        IsDefault: true,
    }
    client.PushNotifications.Credentials.Update(
        context.TODO(),
        "comms_credential_01h9krwprkeee8fzqspvwy6nq8",
        request,
    )
}

File: comms/push-notifications/credentials/update-credential/update-credential.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;
import com.twilio.sdk.resources.pushnotifications.credentials.requests.CredentialsUpdateRequest;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN")
            .build();

        client.pushNotifications().credentials().update(
            "comms_credential_01h9krwprkeee8fzqspvwy6nq8",
            CredentialsUpdateRequest
                .builder()
                .isDefault(true)
                .build()
        );
    }
}